.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode/settings.json
 .build*
+build
```

Comment:
Are you using a different build dir?

Reply:
I looked into this and realized that the IDE I use defaults to using an in-tree build directory. We can remove this particular commit